README.md

```diff
@@ -1137,7 +1138,7 @@ AAAI Conf. Artificial Intelligence, 2021.
 - <a name="Wang21iros"></a>Wang, Z., Pan, L., Ng, Y., Zhuang, Z., Mahony., R.,   
 *[Stereo Hybrid Event-Frame (SHEF) Cameras for 3D Perception](https://doi.org/10.1109/IROS51168.2021.9636312)*,  
 IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2021.
-[PDF](https://arxiv.org/abs/2110.04988), [Video](https://www.youtube.com/watch?v=Azu7rJSPGNc), [Dataset](https://github.com/ziweiWWANG/SHEF)
+[PDF](https://ieeexplore.ieee.org/abstract/document/9636312), [arXiv](https://arxiv.org/abs/2110.04988), [Video](https://www.youtube.com/watch?v=Azu7rJSPGNc), [Dataset](https://github.com/ziweiWWANG/SHEF)
```

Comment:
No need the "PDF" link here since it is the same as the one in the title. 
Please use "PDF" to provide the arXiv link ( https://arxiv.org/pdf/2110.04988 ) so that people without a subscription can still download it.

Reply:
Sounds good! Done!